lib/msf/core/post/windows/version.rb

```diff
@@ -32,15 +32,15 @@ def get_version_info
   end
 
   def get_version_info_fallback_impl
-    build_num_raw = cmd_exec('ver')
+    build_num_raw = cmd_exec('cmd.exe /c ver')
     groups = build_num_raw.match(/.*Version\s+(\d+)\.(\d+)\.(\d+)(?:\.(\d+))?/)
```

Comment:
since you're touching this function:

```suggestion
    groups = build_num_raw.match(/Version\s+(\d+)\.(\d+)\.(\d+)(?:\.(\d+))?/)
```